doc/make.jl

```diff
@@ -162,13 +139,13 @@ const PAGES = [
     ],
 ]
 
-using DelimitedFiles, Test, Mmap, SharedArrays, Profile, Base64, FileWatching, CRC32c,
-      Dates, IterativeEigensolvers, Unicode, Distributed, Printf
+for stdlib in STDLIB_DOCS
+    @eval using $(stdlib.stdlib)
+end
 
 makedocs(
     build     = joinpath(pwd(), "_build/html/en"),
-    modules   = [Base, Core, BuildSysImg, DelimitedFiles, Test, Mmap, SharedArrays, Profile,
-                 Base64, FileWatching, Dates, IterativeEigensolvers, Unicode, Distributed, Printf],
+    modules   = [Base, Core, BuildSysImg, [Module(stdlib.stdlib) for stdlib in STDLIB_DOCS]...],
```

Comment:
Does this work? I thought `Module(x)` made a new empty module.

Great change though :+1:

Reply:
> I thought `Module(x)` made a new empty module.

Well, clearly you haven't been programming Julia long enough...

It works in the sense that the docs render correctly as far as I can see.

Reply:
That's weird; something is up. `stdlib.stdlib` is a Symbol right?

```
julia> using Unicode

julia> Module(:Unicode) === Unicode
false
```

Reply:
Ok, it is wrong but the reason it still work, I think, is that none of the stdlib modules have any submodules, which is what Documenter uses this information. Will correct.